### PR TITLE
Update metrics-template.md

### PR DESCRIPTION
**Pull request by germonprez**
Signed-off-by: Matt Germonprez <germonprez@gmail.com>

@klumb adding the disclaimer to the template

### CODE REVIEW

File: resources/metrics-template.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The following is a draft for the template that all 'Metric Detail' pages can fol
 ```markdown
 # {Name of Metric}
 
+### This metric is a release candidate. To comment on this metric please see Issue #[put the respective Issue Number here]. Following a comment period, this metric will be included in the next regular release.
```
Comment (GeorgLink) [inline resources/metrics-template.md:8]:
```suggestion
### This metric is a release candidate. To comment on this metric please see Issue [#[put the respective Issue Number here]](URL to issue). Following a comment period, this metric will be included in the next regular release.
```

**Review by GeorgLink — CHANGES_REQUESTED** [review]
I suggest linking the issue number to the issue to make it easier for visitors to find the right issue and not have to guess which repository the issue is in.

**Comment by klumb** [comment]
> I suggest linking the issue number to the issue to make it easier for visitors to find the right issue and not have to guess which repository the issue is in.

Yes, thank you georg

**Comment by klumb** [comment]
this looks good I am going to merge this as we have other metrics template PRs coming

**Comment by GeorgLink** [comment]
fixed merge conflict in browser -- breaks DCO sign-off -- manually overwriting and using squash merge.

**Review by GeorgLink — APPROVED** [review]

**Pull request merged**